sdk/identity/Azure.Identity/tests/Data/mock-arc-mi-key.key

```diff
@@ -1 +1 @@
-﻿FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
+﻿FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
```

Comment:
This has a line break that was causing issues in the transport layer.

Reply:
what issues?